src/stars.css

```diff
@@ -0,0 +1,18 @@
+.stars__star {
+  background: url("./images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 > div:first-child,
+.stars--2 > div:nth-child(-n+2),
+.stars--3 > div:nth-child(-n+3),
+.stars--4 > div:nth-child(-n+4),
+.stars--5 > .star {
+  background: url("./images/star-active.svg");
+  background-position: center;
+  background-repeat: no-repeat;
```

Comment:
Delete background-position and background-repeat properties, you've already set it for stars__star class